shadows/framework/src/main/java/org/robolectric/shadows/ShadowApplicationPackageManager.java

```diff
@@ -926,7 +934,8 @@ protected List<ResolveInfo> queryBroadcastReceivers(Intent intent, int flags) {
 
   @Implementation(minSdk = TIRAMISU)
   protected List<ResolveInfo> queryBroadcastReceivers(
-      /*Intent*/ Object intent, /*ResolveInfoFlags*/ Object flags) {
+      Intent intent,
+      @ClassName("android.content.pm.PackageManager$ResolveInfoFlags") Object flags) {
     return queryBroadcastReceivers((Intent) intent, (int) ((ResolveInfoFlags) flags).getValue());
```

Comment:
You can get rid of the casts to (Intent) and (int) (and below too). You still need casts when casting from Object, though.

Reply:
done. also checked other places to remove redundant cast